frontend/src/components/Rating/Review.jsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import { FaStar } from "react-icons/fa";
+import { useState } from "react";
+import "./Review.css";
+function Review({ mealId }) {
+  const [rating, setRating] = useState(null);
+  const [hover, setHover] = useState(null);
+  let currentRating = 1;
+  const [newReview, setNewReview] = useState({
+    stars: "",
+    meal_id: "",
+    create_date: "",
+    title: "",
+  });
+
+  const clickHandler = (event) => {
+    event.preventDefault();
+
+    const starValue = Number(event.target.value);
+    setRating(starValue);
+
+    const meal_id = mealId;
+    // const title = data.data[0].title;
+    const create_date = new Date().toISOString().split("T")[0];
+    const reviewData = JSON.stringify({
+      ...newReview,
+      stars: currentRating,
+      meal_id: Number(meal_id),
+      create_date: create_date,
+    //   title: title,
```

Comment:
if it's not use, lets remove it, or you can use a special syntax to comment code that will be needed later like 
// FIXME: "some comment" or 
// TODO: "some comment"

Those two are really common among developers :)